easybuild/easyconfigs/i/intel/intel-2016.00.eb

```diff
@@ -0,0 +1,22 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'intel'
+version = '2016.00'
```

Comment:
`impi` is in version 5.1.2.150 (update 2). Other components of intel toolchain are update1.  So, there is not consistency from Intel side.

This is just maintenance release, changing every time release of toolchain is not what our users want, I think. They want to use latest updated versions of all Intel tools, but in familiar toolchain name (intel-2016.00). What do you think?